hack/graph-util.py

```diff
@@ -60,12 +87,14 @@ def load_nodes(directory, registry, repository):
                     except ValueError as error:
                         raise ValueError('failed to load YAML from {}: {}'.format(path, error))
                     if not meta:
+                        if entry['name'] == '4.0.0-0.9':
```

Comment:
??

Reply:
whoops.  Dropping this debugging stuff...

Reply:
fixed with f2f4a76 -> beb55be.